modules/logger/logger.py

```diff
@@ -99,15 +100,25 @@ def create(cls, name: str, enable_log_to_file: bool) -> "tuple[bool, Logger | No
             file_handler.setFormatter(formatter)
             logger.addHandler(file_handler)
 
-        return True, Logger(cls.__create_key, logger)
+            return True, Logger(
+                cls.__create_key, logger, pathlib.Path(log_directory_path, log_path)
+            )
+
+        return True, Logger(cls.__create_key, logger, None)
```

Comment:
Why is there 2 returns here?

Reply:
`log_path` doesn't exist when `enable_log_to_file` is false.